app/src/main/java/mobileapp/ctemplar/com/ctemplarapp/login/step/StepPasswordFragment.java

```diff
@@ -87,28 +66,23 @@ public void onPause() {
 
     @OnClick({R.id.fragment_step_password_next_btn})
     public void onClickNext() {
-        if (editChoose.length() < 7) {
-            editConfirmLayout.setError(getString(R.string.error_password_small));
+        if (passwordEditText.length() < 7) {
+            passwordInputLayout.setError(getString(R.string.error_password_small));
```

Comment:
There should be no limit on password length

Reply:
Done